src/plugins/visible-range/index.js

```diff
@@ -126,6 +131,36 @@ export default class VisibleRangePlugin extends AbstractPlugin {
     }
   }
 
+  /**
+   * @summary Gets the latitude range defined by the viewer's panoData
+   */
+  getPanoLatitudeRange() {
+    const p = this.psv.prop.panoData;
+    if (p.croppedHeight == p.fullHeight && p.croppedY == 0) {
```

Comment:
this probably does not pass the linter, see the build logs

Reply:
Thanks. :slightly_smiling_face: I saw it. Fixed.